brd-ios/breadwallet/src/ViewControllers/RootNavigationController.swift

```diff
@@ -27,17 +27,11 @@ class RootNavigationController: UINavigationController, UINavigationControllerDe
         let tintColor: UIColor
         
         switch viewController {
-        case is AccountViewController,
-            is HomeScreenViewController,
-            is KYCCameraViewController,
```

Comment:
Are we sure?

Reply:
yes, it's the same color for default case - Background.two

Reply:
@dangelovska 
![image](https://user-images.githubusercontent.com/99484553/198195713-3be1bd91-b5e6-48a1-af6a-883d34500103.png)

Reply:
![image](https://user-images.githubusercontent.com/99484553/198196550-50918359-ee12-495c-90c7-dfcbb76e9075.png)

Reply:
![IMG_2879](https://user-images.githubusercontent.com/99484553/198196820-1546f05f-fdcd-4161-b168-05a81f1078b0.PNG)